docs/src/pages/components/cdk-components/switch/api.jsx

```diff
@@ -9,6 +9,11 @@ const switchPropsTable = () => {
         <th>Default</th>
         <th>Description</th>
       </tr>
+      <tr>
+        <td>defaultChecked: boolean</td>
+        <td></td>
```

Comment:
I have some concerns with this. I don't know if `defaultChecked` or any other `defaultValue` prop should have a default value. I see the value of having it, but, the `defaultValue` react prop that we are trying to reproduce, doesn't work like this, since it is an optional prop whose value is undefined when it is not passed by the user. In this case, It's my fault, in the component, `defaultChecked` should not have value.
Anyways, I'm not completely sure in how to approach this (search for 'defaultChecked' in [Fluent UI](https://developer.microsoft.com/en-us/fluentui#/controls/web/checkbox)).

Reply:
I think `defaultChecked` and `defaultValue` shouldn't have a default value as they are mutually exclusive, as explained in Fluent UI documentation.

I will accept the changes from last commit, I think it is the best approach to avoid confusion.